.github/workflows/sanity.yml

```diff
@@ -20,8 +20,46 @@ jobs:
     name: Sanity
     strategy:
       matrix:
-        php: [ '8.1', '8.2', '8.3' ]
-        browser: [ 'chromium', 'firefox', 'webkit' ]
+        include:
```

Comment:
Hmm, this means we're gonna have 18 jobs for sanity? Isn't it a bit too much to have this on ALL pull requests?

Reply:
Nope... We had 9 jobs before, and now we have 12 jobs, validated with @PrestaShop/qa-functional team. (We avoid to use 18 jobs, but we test the behavior with each version of PHP).

Reply:
Ah ok mariadb is only tested with chrome, nice!